modules/nf-core/pyrodigal/main.nf

```diff
@@ -4,17 +4,17 @@ process PYRODIGAL {
 
     conda "bioconda::pyrodigal=2.1.0"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/pyrodigal:2.1.0--py310h1425a21_0':
-        'quay.io/biocontainers/pyrodigal:2.1.0--py310h1425a21_0' }"
+        'https://depot.galaxyproject.org/singularity/mulled-v2-2fe9a8ce513c91df34b43a6610df94c3a2eb3bd0:697b3838b186fac6a9ceec198b09d4032162a079-0':
+        'biocontainers/mulled-v2-2fe9a8ce513c91df34b43a6610df94c3a2eb3bd0:697b3838b186fac6a9ceec198b09d4032162a079-0' }"
 
     input:
     tuple val(meta), path(fasta)
 
     output:
-    tuple val(meta), path("*.gff")      , emit: gff
-    tuple val(meta), path("*.fna")      , emit: fna
-    tuple val(meta), path("*.faa")      , emit: faa
-    tuple val(meta), path("*.score")    , emit: score
+    tuple val(meta), path("*.gff.gz")      , emit: gff
+    tuple val(meta), path("*.fna.gz")      , emit: fna
+    tuple val(meta), path("*.faa.gz")      , emit: faa
+    tuple val(meta), path("*.score.gz")    , emit: score
```

Comment:
This change isn't in the changelog?

Reply:
And can you confirm we don't publishg the files from here if `--save_annotation`, but rather from the downstream `gunzip`?

Reply:
We publish these `.gz` files from the module, not the `gunzip`-ed files. That is the intended behaviour, right?
But I noticed some relevant typos in modules.config, will push the correction in this PR.

Reply:
And this change is included in the changelog. I summarised it in PR #274 but didn't explicitly mention the `.gz` changes. Will update that as well!